synapse/storage/databases/main/relations.py

```diff
@@ -63,6 +64,11 @@ def __init__(
         self._msc1849_enabled = hs.config.experimental.msc1849_enabled
         self._msc3440_enabled = hs.config.experimental.msc3440_enabled
 
+        self.get_applicable_edit: LruCache[str, Optional[EventBase]] = LruCache(
```

Comment:
Not sure about the name---`get_applicable_edit` sounds like a method to me. `applicable_edit_cache` maybe?

Reply:
Ah, I did this to reduce code-churn, but definitely should be cleaned up. Thanks for the prompt!